src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -59,7 +59,7 @@ def _main(settings, optlist, args):
         if os.path.exists(results_location):
             log.info("Uploading results from {}".format(results_location))
 
-            with open(results_location, 'r') as result_file:
+            with open(results_location, encoding="UTF-8") as result_file:
```

Comment:
I doubt it matters much but you've removed the read-only-ness of this.

Reply:
By default the file is opened for reading only so `'r'` is not necessary

https://docs.python.org/3/library/functions.html#open